app/src/main/java/org/oppia/android/app/administratorcontrols/ShowLogoutDialogListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.administratorcontrols
+
+/** Listener for when an activity should show [LogoutDialogFragment]. */
```

Comment:
Suggest: ``Listener for when the logout dialog should be shown.``

Reply:
Done